code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1326,11 +1327,11 @@
 
 /datum/reagent/medicine/tricordrazine
 	name = "Tricordrazine"
-	description = "Has a high chance to heal all types of damage. Overdose causes toxin damage and liver damage."
+	description = "Heals all types of damage based on damage types you have and its severity. Overdose causes toxin damage and liver damage."
 	reagent_state = LIQUID
 	color = "#707A00" //tricord's component chems mixed together, olive.
 	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_FUN | CHEMICAL_RNG_BOTANY
-	metabolization_rate = 3 * REAGENTS_METABOLISM
+	metabolization_rate = 1.5 * REAGENTS_METABOLISM
```

Comment:
I think considering tricordrazine is a little harder to make the healing should be better.
It has 1/3 the effectiveness when compared to the other basic chemicals due to the high metabolisation rate but still low healing amount.
I think a heal value of ~2.5 or a metabolisation rate of 0.6 would be an improvement, since then its only slightly worse than the basic chems but has the increased speed and healing all of them.